src/test/store/zipped-profiles.test.js

```diff
@@ -67,7 +67,7 @@ describe('reducer zipFileState', function() {
     it('computes a profile name from the loaded file', async function() {
       const { getState } = await setup();
 
-      const expectedName = 'profile1.json';
+      const expectedName = 'bar/profile1.json';
       expect(UrlStateSelectors.getProfileName(getState())).toBe(expectedName);
     });
```

Comment:
do you think you can add a few more test cases to test more cases ?

Here is a suggestion:
1. add to the code with `storeWithZipFile` a line with no directory at all  (eg: `profile4.json` without any directory).
2. in this test, do more dispatches with:
```js
     await dispatch(
        ZippedProfilesActions.viewProfileFromPathInZipFile(
          'foo/bar/profile1.json'
        )
      );
```

but replace the actual zip file by some other values, and check the computed profile name again.

Reply:
Will add some and update within a few hours

Reply:
@julienw Let me know if changes are needed in this test or what other tests could be added.

Reply:
I think you can do it in the same test, by dispatching `returnToZipFileList` also between.